test/suites/dev/moonbase/test-xcm-v4/test-xcm-evm-with-dot.ts

```diff
@@ -0,0 +1,162 @@
+        supply: balance,
+      });
+
+      // Fund descend address with enough xcDOTs to pay XCM message and EVM execution fees
+      await mockAssetBalance(context, assetBalance, assetDetails, alith, assetId, descendAddress);
+
+      // Deploy example contract to be called through XCM
+      const { contractAddress, abi } = await context.deployContract!("Incrementor");
+      contractABI = abi;
+      contractDeployed = contractAddress;
+
+      await verifyLatestBlockFees(context);
+    });
+
+    it({
+      id: "T01",
+      title: "should execute EVM remote call through XCM paying fees in DOT",
+      test: async function () {
+        // Since we cannot infer the actual weitght of the inner message,
+        // we are using big enough gas limits to be able to execute the whole xcm transaction.
+        const xcmTransaction = {
+          V1: {
+            gas_limit: 155_000,
+            fee_payment: {
+              Auto: {
+                Low: null,
+              },
+            },
+            action: {
+              Call: contractDeployed,
+            },
+            value: 0,
+            input: encodeFunctionData({
+              abi: contractABI,
+              functionName: "incr",
+              args: [],
+            }),
+            access_list: null,
+          },
+        };
+
+        const XCDOT_FEE_AMOUNT = 100_000n;
+
+        const config: XcmFragmentConfig = {
+          assets: [
+            {
+              // We refer to DOT here, because XCM will internally convert it to xcDOT
+              multilocation: {
+                parents: 1,
+                interior: { Here: null },
+              },
+              fungible: XCDOT_FEE_AMOUNT,
+            },
+          ],
+          weight_limit: {
+            refTime: 100_000_000_000,
+            proofSize: 80_000,
+          } as any,
+          descend_origin: sendingAddress,
+          beneficiary: sendingAddress,
+        };
+
+        const transferCall = api.tx.ethereumXcm.transact(xcmTransaction);
+        const transferCallEncoded = transferCall?.method.toHex();
+
+        // Build XCM transaction with remote EVM call
+        const xcmMessage = new XcmFragment(config)
+          .descend_origin()
+          .withdraw_asset()
+          .buy_execution()
+          .push_any({
+            Transact: {
+              originKind: "SovereignAccount",
+              requireWeightAtMost: {
+                refTime: 50_000_000_000,
+                proofSize: 50_000,
+              },
+              call: {
+                Call: transferCallEncoded,
+              },
+            },
+          })
+          .as_v3();
+
+        // Send an XCM and create block to execute it
+        await injectHrmpMessageAndSeal(context, 1, {
+          type: "XcmVersionedXcm",
+          payload: xcmMessage,
+        } as RawXcmMessage);
+
+        const resultingAlithBalance = (await api.query.assets.account(assetId, descendAddress))
```

Comment:
This is not Alith balance, Alith is the sudo account.
```suggestion
        const resultingSenderBalance = (await api.query.assets.account(assetId, descendAddress))
```